armory/data/datasets.py

```diff
@@ -1,5 +1,9 @@
 """
 Download and preprocess common datasets.
+Each standard dataset resides in its own subdirectory under dataset_dir based
+upon the name of the function in the datasets file.
+The download subdirectory under datset_dir is reserved for caching.
```

Comment:
nit: datset_dir is misspelled. 

Also, add quotes or ticks around the specific subdirectory names. 

Add an example to the standard dataset, (e.g., "cifar10")